src/pages/IstioConfigDetails/IstioObjectDetails/VirtualServiceDetail.tsx

```diff
@@ -94,23 +94,31 @@ class VirtualServiceDetail extends React.Component<VirtualServiceProps> {
           <CardBody>
             <Text component={TextVariants.h2}>Virtual Service Overview</Text>
             {globalStatus}
-            <Text component={TextVariants.h3}>Created at</Text>
-            <LocalTime time={virtualService.metadata.creationTimestamp || ''} />
-
-            <Text component={TextVariants.h3}>Resource Version</Text>
-            {virtualService.metadata.resourceVersion}
-
-            {virtualService.spec.hosts && virtualService.spec.hosts.length > 0 ? (
-              <>
-                <Text component={TextVariants.h3}>Hosts</Text>
-                <DetailObject name="" detail={virtualService.spec.hosts} validation={this.hostStatusMessage()} />
-              </>
-            ) : (
-              undefined
-            )}
-            {virtualService.spec.gateways && virtualService.spec.gateways.length > 0
-              ? this.generateGatewaysList(virtualService.spec.gateways, isValid)
-              : undefined}
+            <Stack>
+              <StackItem id={'created_at'}>
+                <Text component={TextVariants.h3}>Created at</Text>
+                <LocalTime time={virtualService.metadata.creationTimestamp || ''} />
+              </StackItem>
+              <StackItem id={'resource_version'}>
+                <Text component={TextVariants.h3}>Resource Version</Text>
+                {virtualService.metadata.resourceVersion}
+              </StackItem>
+              <StackItem id={'hosts'}>
+                {virtualService.spec.hosts && virtualService.spec.hosts.length > 0 ? (
+                  <>
+                    <Text component={TextVariants.h3}>Hosts</Text>
```

Comment:
Should this StackItem go right between the `<>` and `<Text>` components? Otherwise we will have tags that are no needed? Does it make sense?

Reply:
I was thinking not to complicate code readability, as in a case of VS Gateways, it will be over-complication, and an empty tag does not have impact on the UI.